test/multiverse/suites/active_record/Envfile

```diff
@@ -90,10 +90,12 @@ end
 # Only 3.2.22 fixes the above mentioned issue, so it's the only one 
 # surviving in our coverage of Rails 3.x series here.
 #
-if RUBY_VERSION <= '2.2.0'
+if RUBY_VERSION < '2.3'
+  mysql_vsn = '0.3.20'
```

Comment:
I'm not sure if 0.3.21 gives us anything we can leverage, but it's the version I've been testing with (latest stable 0.3.x)